index.js

```diff
@@ -120,10 +120,14 @@ class HandlebarsPlugin {
         if (compiler.hooks) {
             // @feature html-webpack-plugin
             if (this.options.htmlWebpackPlugin.enabled) {
-                const { prefix } = this.options.htmlWebpackPlugin;
-
                 compiler.hooks.compilation.tap("HtmlWebpackPluginHooks", compilation => {
-                    compilation.hooks.htmlWebpackPluginAfterHtmlProcessing.tap("HandlebarsRenderPlugin", this.processHtml.bind(this));
+                    // html-webpack-plugin < 4
+                    if (compilation.hooks.htmlWebpackPluginAfterHtmlProcessing) {
+                        compilation.hooks.htmlWebpackPluginAfterHtmlProcessing.tap("HandlebarsRenderPlugin", this.processHtml.bind(this));
+                    } else if (HtmlWebpackPlugin.getHooks) {
```

Comment:
you are using safe-require, which returns `undefined` for the missing module. This conditional will throw an Error, when the module is not found.